frontend/src/components/common/GenericBreadCrumb.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import {
+    Breadcrumb,
+    BreadcrumbItem,
+    Grid,
+    Column,
+  } from "@carbon/react";
+  import { useIntl } from "react-intl";
+  import { Link, useLocation } from 'react-router-dom';
+
+  const GenericHomeBreadCrumb = () => {
+    const intl = useIntl();
+
+    
+const Breadcrumbs = ({ title, breadcrumbs }) => {
+  return (
+    <Grid fullWidth={true}>
+    <Column lg={16}>
+      <Breadcrumb>
+        <BreadcrumbItem href="/">
+
+        {breadcrumbs.map((crumb, index) => (
+        <span key={index}>
+          {index < breadcrumbs.length - 1 ? (
+            <Link to={crumb.link}>{crumb.label}</Link>
+          ) : (
+            <span>{crumb.label}</span>
+          )}
+          {index < breadcrumbs.length - 1 && '>'}
+        </span>
+      ))}
+
+        </BreadcrumbItem>
+      </Breadcrumb>
+    </Column>
+   </Grid>
+  );
+};
+
+const breadcrumbData = [
+  { path: '/', title: intl.formatMessage({ id: "home.label" }) },
+  { path: '/ResultValidation', title: intl.formatMessage({ id: "sidenav.label.validation" }) }
```

Comment:
why was this hard coded here??

Reply:
The  `breadcrumbData` will pass on all these eventually to the path prop `<BreadCrumbItem href={path}>` and {title} props